TACC/Core-Styles
# hotfix: tup-660 increase cms form badge space

## Overview

Increase space before a Django CMS form field "(required)" badge.

## Related

- [TUP-660](https://jira.tacc.utexas.edu/browse/TUP-660)
- follow up to #269

## Changes

- **added** more form badge space for django cms form only

## Testing

1. Crreate/Open a form built with Django CMS form plugin.
2. Add/View a required field.
3. Verify space between label text and field is the width of an "m".

## UI

Skipped.
